samples/visual-indicator/nodejs/README.md

```diff
@@ -0,0 +1,128 @@
+---
+page_type: sample
+description: This sample app shows how to send visual indicator in meetings using targeted notifications API.
```

Comment:
This sample app shows how to send visual indicator in Teams meetings using targeted notifications API.